### PR TITLE
Moves to GitHub actions build (#188)

### PR DESCRIPTION
**Pull request by codefromthecrypt**
* Travis is still cutoff despite over a week of email.
 * We have to switch to GitHub actions to avoid paralyzing the project

As a part of this, the build is refactored to use common scripts,
documented in build-bin/README.md

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request
+about: Please first, look at existing issues to see if the feature has been requested before.
+labels: enhancement
+---
+Please first, look at [existing issues](https://github.com/openzipkin/zipkin-gcp/issues) to see if the feature has been requested before. If you don't find anything tell us what problem you’re trying to solve. Often a solution already exists! Don’t send pull requests to implement new features without first getting our support. Sometimes we leave features out on purpose to keep the project small.
```
Comment (codefromthecrypt) [inline .github/ISSUE_TEMPLATE/feature.md]:
```suggestion
Please first, look at [existing issues](https://github.com/openzipkin/brave-karaf/issues) to see if the feature has been requested before. If you don't find anything tell us what problem you’re trying to solve. Often a solution already exists! Don’t send pull requests to implement new features without first getting our support. Sometimes we leave features out on purpose to keep the project small.
```

File: pom.xml [hunk 2]
```diff
@@ -29,29 +29,63 @@
...
+    </snapshotRepository>
+    <repository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
+
+  <issueManagement>
+    <system>Github</system>
+    <url>https://github.com/openzipkin/zipkin-gcp/issues</url>
```
Comment (codefromthecrypt) [inline pom.xml]:
```suggestion
    <url>https://github.com/openzipkin/brave-karaf/issues</url>
```

**Review by reta — APPROVED** [review]
Super-clean docs @adriancole !

**Comment by codefromthecrypt** [comment]
thanks for the look @reta

**Pull request merged**